wiki/Tournaments/GBC/GBC_2022_Autumn/en.md

```diff
@@ -173,6 +174,59 @@ The GB Cup 2022 Autumn is run by the Team GB and various community members.
 
 ## Match results
 
+### Reugular stage summary
+
+| #[^rs-summary-seed] | Teams | abbr.[^rs-summary-abbr] | MR[^rs-summary-mr] | BR[^rs-summary-br] | BD[^rs-summary-bd] | BU[^rs-summary-bu] | Round 1 | Round 2 | Round 3 | Round 4 |
```

Comment:
also the column numbers don't match. if you need to include the last one, fill it with blank cells.

```suggestion
| #[^rs-summary-seed] | Team | Abbr. | MR[^rs-summary-mr] | BR[^rs-summary-br] | BD[^rs-summary-bd] | BU[^rs-summary-bu] | Round 1 | Round 2 | Round 3 | Round 4 |
```